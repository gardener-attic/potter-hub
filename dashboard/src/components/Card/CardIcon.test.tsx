import { shallow } from "enzyme";
import * as React from "react";

import CardIcon from "./CardIcon";

it("should return an empty object if the icon is not set", () => {
  const wrapper = shallow(<CardIcon />);
  expect(wrapper.html()).toBe(null);
});

it("should render the icon if set", () => {
  const wrapper = shallow(<CardIcon icon="foo" />);
  expect(wrapper.html()).not.toBe(null);
  expect(wrapper.find("img").props().src).toBe("foo");
  expect(wrapper).toMatchSnapshot();
});
