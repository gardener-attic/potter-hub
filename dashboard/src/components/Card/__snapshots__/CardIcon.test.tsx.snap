// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the icon if set 1`] = `
<div
  className="Card__icon bg-light text-c"
>
  <img
    alt=""
    src="foo"
  />
</div>
`;
